src/extension/logger.js

```diff
@@ -0,0 +1,11 @@
+import _ from 'underscore';
+
+export async function info(msg, object) {
+  if (isVerbose()) {
```

Comment:
why are you not using one of the existing modules for logging (winston or something else)?